lxd/storage/drivers/driver_zfs.go

```diff
@@ -271,7 +271,7 @@ func (d *zfs) Create() error {
 		}
 
 		// Create the zpool.
-		_, err = shared.RunCommand("zpool", "create", "-m", "none", "-O", "compression=on", d.config["zfs.pool_name"], loopPath)
+		_, err = shared.RunCommand("zpool", "create", "-m", "none", "-O", "compression=on", "-o", "feature@async_destroy=disabled", d.config["zfs.pool_name"], loopPath)
```

Comment:
Can we combine the common args into a slice used in all places rather than dupe them?